src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -779,6 +779,7 @@ override Microsoft.CodeAnalysis.Semantics.OperationWalker.VisitWhileUntilLoopSta
 override Microsoft.CodeAnalysis.Semantics.OperationWalker.VisitWithStatement(Microsoft.CodeAnalysis.Semantics.IWithStatement operation) -> void
 override Microsoft.CodeAnalysis.Semantics.OperationWalker.VisitYieldBreakStatement(Microsoft.CodeAnalysis.Semantics.IReturnStatement operation) -> void
 override Microsoft.CodeAnalysis.SyntaxNode.ToString() -> string
+static Microsoft.CodeAnalysis.CaseInsensitiveComparison.StartsWith(string value, string possibleStart) -> bool
```

Comment:
Seems perfectly reasonable API to have.

---
In reply to: [76499645](https://github.com/dotnet/roslyn/pull/13415#discussion_r76499645) [](ancestors = 76499645)